### PR TITLE
Added documentation for Maximum Or Minimum Stack Implementation

### LINKED ISSUE #7757: [Create New Issue]: Implementing Min/Max Stack
### Description

Implementation of a stack class which will support the following functions : 
push()
pop()
get_max()

### Domain

DSA

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by VaibhavMalik4187**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #7757

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [X] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [X] DSA

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by VaibhavMalik4187** [comment]
@nimisha-yadav Can you please review my PR?

**Review by nimisha-yadav — COMMENTED** [review]
@VaibhavMalik4187 please add your file directly in [this folder](https://github.com/girlscript/winter-of-contributing/tree/DSA/DSA/4.1%20Stacks/Standard%20Problems)

File: DSA/4.1 Stacks/17. Minimum Or Maximum Stack Implementation/MinimumOrMaximumStack.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+<br>
+
+## **Output**
+
+<br>
+
+![image](https://user-images.githubusercontent.com/34866732/142176253-b8ddc9ab-a7e5-4f71-a5a3-3994e5b21662.png)
+
+<br>
```
Comment (nimisha-yadav) [inline DSA/4.1 Stacks/17. Minimum Or Maximum Stack Implementation/MinimumOrMaximumStack.md:230]: Add time and space complexities as well

Comment (VaibhavMalik4187) [reply DSA/4.1 Stacks/17. Minimum Or Maximum Stack Implementation/MinimumOrMaximumStack.md]: @nimisha-yadav I have made all the changes that were asked by you, can you please review my PR again?

**Review by nimisha-yadav — APPROVED** [review]

**Pull request merged**